client/src/App.js

```diff
@@ -42,11 +154,32 @@ function App() {
             <Home />
           </Route>
           <Route path="/market">
-            {markets.map((market) => (
-              <MarketCard market={market} onAddComment={addComment} />
+            <SearchboxInput
+              type="text"
+              placeholder="Suche hier deinen Markt."
+              onChange={searchedMarkets}
+            />
+            {filteredMarkets &&
+              filteredMarkets.map((filteredMarket) => (
+                <MarketCard
+                  market={filteredMarket}
+                  onAddComment={addComment}
+                  onAddRating={addRating}
+                  onAddToFav={toggleFav}
+                  isFavorite={isFavorite}
+                />
+              ))}
```

Comment:
Could be extracted to a page to keep the `App.js` smaller and hide the details within that component/page